ui/src/app/machines/views/MachineList/PowerColumn/PowerColumn.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import classNames from "classnames";
+import { Loader } from "@canonical/react-components";
+import { useSelector } from "react-redux";
+
+import { machine as machineSelectors } from "app/base/selectors";
+
+const PowerColumn = ({ systemId }) => {
+  const machine = useSelector(state =>
+    machineSelectors.getBySystemId(state, systemId)
+  );
+
+  const iconClass = classNames({
+    "p-icon--power-on": machine.power_state === "on",
+    "p-icon--power-off": machine.power_state === "off",
+    "p-icon--power-error": machine.power_state === "error",
+    "p-icon--power-unknown": machine.power_state === "unknown"
+  });
+
+  if (machine.powerTransition) {
```

Comment:
`machine.powerTransition` is actually not part of the machine object returned from the websocket - it was chucked on as part of the angular code. So for now this won't do anything.

Reply:
Ah thanks @Caleb-Ellis quite right, it is indeed only in response to an action. I think we'd probably handle this differently in redux, rather than tacking on a property to the machine model, so I'm just going to remove this for now until we implement actions.